addons/calendar/models/calendar_event.py

```diff
@@ -674,10 +674,26 @@ def write(self, values):
 
     @api.model_create_multi
     def create(self, vals_list):
-        vals_list = [  # Else bug with quick_create when we are filter on an other user
-            dict(vals, user_id=self.env.user.id) if not 'user_id' in vals else vals
-            for vals in vals_list
-        ]
+        new_vals_list = []
+
+        for vals in vals_list:
+            new_vals = vals
+
+            # Add the current user to the data
+            # Else bug with quick_create when we are filter on an other user
+            if not 'user_id' in vals:
+                new_vals = dict(new_vals, user_id=self.env.user.id)
+            
+            # Add the current user in the attendees if there is none.
+            # It can happen when the user quickcreate an event, which leads to inconstitancy between what we 
+            # create and the result. Indeed, when we use the quickcreate the actual user is not passed in the 
+            # data, so there will be no invitation sent and the status will no be correctly updated since the 
+            # attendee will be added by the default after this method.
+            if not 'partner_ids' in vals:
+                new_vals = dict(new_vals, partner_ids=[[6, False, [self.env.user.partner_id.id]]])
+            
+            new_vals_list.append(new_vals)
+        vals_list = new_vals_list
```

Comment:
Ok next set of questions then:
- Does the problem only occur in "master" and not on a stable branch?
- Why use "6, False, ..." instead of "4, ..." ?
- Why not implement the fix the same way as the "user_id" one for a much smaller diff?
- Do we really want the current user to receive an email invitation for his own meeting? Is that intended?
- What is the problem with the attendee status?

Sorry to be picky about this but modifying the "create" method should always be done carefully as it could introduce more weird behaviors.